// epic-free-games/dist/common/validate.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.isDate = exports.isArray = exports.isBoolean = exports.isNumber = exports.isString = exports.isObject = exports.isNull = void 0;
/**
 * Simple is null check
 */
function isNull(value) {
    return value === null;
}
exports.isNull = isNull;
/**
 * Simple is object check
 */
const isObject = (value) => value ? typeof value === 'object' && !Array.isArray(value) && !isNull(value) : false;
exports.isObject = isObject;
/**
 * Simple is string check
 */
function isString(value) {
    return typeof value === 'string';
}
exports.isString = isString;
/**
 * Simple is number check
 */
function isNumber(value) {
    return typeof value === 'number';
}
exports.isNumber = isNumber;
/**
 * Simple is boolean check
 */
function isBoolean(value) {
    return typeof value === 'boolean';
}
exports.isBoolean = isBoolean;
/**
 * Simple is array check
 */
function isArray(value) {
    return Array.isArray(value);
}
exports.isArray = isArray;
/**
 * Simple is date check
 * https://stackoverflow.com/a/44198641
 */
function isDate(value) {
    if (!value)
        return false;
    return Object.prototype.toString.call(value) === '[object Date]' && !Number.isNaN(value);
}
exports.isDate = isDate;
