// epic-free-games/dist/modules/epic-free-games/test-data.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.testData = void 0;
exports.testData = {
    current: {
        valid: {
            title: 'Beyond Blue',
            id: 'c3006006e4c74c4d8316ff09fb8dcc8d',
            namespace: 'bd9fd28ca1214868922f2ab41d6d3b65',
            description: 'Beyond Blue is a single-player narrative adventure that takes you deep into our planet’s beating blue heart.  Set in the near future, Beyond Blue explores the mysteries of our ocean through the eyes of Mirai, a deep-sea explorer and scientist. Become a part of a newly-formed r...',
            effectiveDate: '2020-06-05T09:59:00.000Z',
            offerType: 'BASE_GAME',
            expiryDate: null,
            viewableDate: null,
            status: 'ACTIVE',
            isCodeRedemptionOnly: false,
            keyImages: [
                {
                    type: 'DieselStoreFrontWide',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S1-2560x1440-5d205f1273f6b805db82bc525a1432b9.jpg'
                },
                {
                    type: 'DieselStoreFrontTall',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'OfferImageWide',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S1-2560x1440-5d205f1273f6b805db82bc525a1432b9.jpg'
                },
                {
                    type: 'OfferImageTall',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'Thumbnail',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'CodeRedemption_340x440',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'DieselGameBox',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S1-2560x1440-5d205f1273f6b805db82bc525a1432b9.jpg'
                }
            ],
            seller: {
                id: 'o-4ngf8f445bswz3vfhmvbfxbgf67bc4',
                name: 'E-Line Media'
            },
            productSlug: 'beyond-blue',
            urlSlug: 'beyond-blue',
            url: null,
            items: [
                {
                    id: 'def4268a97694dfb85572ea4990d86ce',
                    namespace: 'bd9fd28ca1214868922f2ab41d6d3b65'
                }
            ],
            customAttributes: [
                {
                    key: 'com.epicgames.app.blacklist',
                    value: '[]'
                },
                {
                    key: 'com.epicgames.app.productSlug',
                    value: 'beyond-blue'
                }
            ],
            categories: [
                {
                    path: 'freegames'
                },
                {
                    path: 'games'
                },
                {
                    path: 'games/edition'
                },
                {
                    path: 'games/edition/base'
                },
                {
                    path: 'applications'
                }
            ],
            tags: [
                {
                    id: '1296'
                },
                {
                    id: '21894'
                },
                {
                    id: '1370'
                },
                {
                    id: '9547'
                },
                {
                    id: '1117'
                },
                {
                    id: '1263'
                }
            ],
            catalogNs: {
                mappings: [
                    {
                        pageSlug: 'beyond-blue',
                        pageType: 'productHome'
                    }
                ]
            },
            offerMappings: [],
            price: {
                totalPrice: {
                    discountPrice: 0,
                    originalPrice: 1999,
                    voucherDiscount: 0,
                    discount: 1999,
                    currencyCode: 'USD',
                    currencyInfo: {
                        decimals: 2
                    },
                    fmtPrice: {
                        originalPrice: '$19.99',
                        discountPrice: '0',
                        intermediatePrice: '0'
                    }
                },
                lineOffers: [
                    {
                        appliedRules: [
                            {
                                id: 'e5c5f6e73c4c410a925f239061e61c68',
                                endDate: '2023-04-27T15:00:00.000Z',
                                discountSetting: {
                                    discountType: 'PERCENTAGE'
                                }
                            }
                        ]
                    }
                ]
            },
            promotions: {
                promotionalOffers: [
                    {
                        promotionalOffers: [
                            {
                                startDate: '2023-04-20T15:00:00.000Z',
                                endDate: '2023-04-27T15:00:00.000Z',
                                discountSetting: {
                                    discountType: 'PERCENTAGE',
                                    discountPercentage: 0
                                }
                            }
                        ]
                    }
                ],
                upcomingPromotionalOffers: []
            }
        },
        invalid: {
            someRandomKey: 'xd'
        },
        other: {
            title: 'Beyond Blue',
            id: 'c3006006e4c74c4d8316ff09fb8dcc8d',
            namespace: 'bd9fd28ca1214868922f2ab41d6d3b65',
            description: 'Beyond Blue is a single-player narrative adventure that takes you deep into our planet’s beating blue heart.  Set in the near future, Beyond Blue explores the mysteries of our ocean through the eyes of Mirai, a deep-sea explorer and scientist. Become a part of a newly-formed r...',
            effectiveDate: '2020-06-05T09:59:00.000Z',
            offerType: 'OTHERS',
            expiryDate: null,
            viewableDate: null,
            status: 'ACTIVE',
            isCodeRedemptionOnly: false,
            keyImages: [
                {
                    type: 'DieselStoreFrontWide',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S1-2560x1440-5d205f1273f6b805db82bc525a1432b9.jpg'
                },
                {
                    type: 'DieselStoreFrontTall',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'OfferImageWide',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S1-2560x1440-5d205f1273f6b805db82bc525a1432b9.jpg'
                },
                {
                    type: 'OfferImageTall',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'Thumbnail',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'CodeRedemption_340x440',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S2-1200x1600-66ee2b1322a74ff2e35e462d644ff519.jpg'
                },
                {
                    type: 'DieselGameBox',
                    url: 'https://cdn1.epicgames.com/bd9fd28ca1214868922f2ab41d6d3b65/offer/EGS_BeyondBlue_ELineMedia_S1-2560x1440-5d205f1273f6b805db82bc525a1432b9.jpg'
                }
            ],
            seller: {
                id: 'o-4ngf8f445bswz3vfhmvbfxbgf67bc4',
                name: 'E-Line Media'
            },
            productSlug: 'beyond-blue',
            urlSlug: 'beyond-blue',
            url: null,
            items: [
                {
                    id: 'def4268a97694dfb85572ea4990d86ce',
                    namespace: 'bd9fd28ca1214868922f2ab41d6d3b65'
                }
            ],
            customAttributes: [
                {
                    key: 'com.epicgames.app.blacklist',
                    value: '[]'
                },
                {
                    key: 'com.epicgames.app.productSlug',
                    value: 'beyond-blue'
                }
            ],
            categories: [
                {
                    path: 'freegames'
                },
                {
                    path: 'games'
                },
                {
                    path: 'games/edition'
                },
                {
                    path: 'games/edition/base'
                },
                {
                    path: 'applications'
                }
            ],
            tags: [
                {
                    id: '1296'
                },
                {
                    id: '21894'
                },
                {
                    id: '1370'
                },
                {
                    id: '9547'
                },
                {
                    id: '1117'
                },
                {
                    id: '1263'
                }
            ],
            catalogNs: {
                mappings: [
                    {
                        pageSlug: 'beyond-blue',
                        pageType: 'productHome'
                    }
                ]
            },
            offerMappings: [],
            price: {
                totalPrice: {
                    discountPrice: 0,
                    originalPrice: 1999,
                    voucherDiscount: 0,
                    discount: 1999,
                    currencyCode: 'USD',
                    currencyInfo: {
                        decimals: 2
                    },
                    fmtPrice: {
                        originalPrice: '$19.99',
                        discountPrice: '0',
                        intermediatePrice: '0'
                    }
                },
                lineOffers: [
                    {
                        appliedRules: [
                            {
                                id: 'e5c5f6e73c4c410a925f239061e61c68',
                                endDate: '2023-04-27T15:00:00.000Z',
                                discountSetting: {
                                    discountType: 'PERCENTAGE'
                                }
                            }
                        ]
                    }
                ]
            },
            promotions: {
                promotionalOffers: [
                    {
                        promotionalOffers: [
                            {
                                startDate: '2023-04-20T15:00:00.000Z',
                                endDate: '2023-04-27T15:00:00.000Z',
                                discountSetting: {
                                    discountType: 'PERCENTAGE',
                                    discountPercentage: 0
                                }
                            }
                        ]
                    }
                ],
                upcomingPromotionalOffers: []
            }
        }
    },
    next: {
        valid: {
            title: 'Breathedge',
            id: '8401414902e84f2cb9afa9142f051d32',
            namespace: '08ae29e4f70a4b62aa055e383381aa82',
            description: 'Survive in outer space! Together with your immortal chicken, discover the truth behind your sudden spaceship crash. Craft tools, pilot vehicles, and even control space stations to survive and explore the wreckage.',
            effectiveDate: '2021-02-25T18:00:00.000Z',
            offerType: 'BASE_GAME',
            expiryDate: null,
            viewableDate: '2021-02-09T17:00:00.000Z',
            status: 'ACTIVE',
            isCodeRedemptionOnly: false,
            keyImages: [
                {
                    type: 'OfferImageWide',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/egs-breathedge-redruinssoftworks-s1-2560x1440-76fd08cd9c2d-2560x1440-783424f6d4a7f32a0e75fc074f2f928e.jpg'
                },
                {
                    type: 'DieselStoreFrontWide',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/egs-breathedge-redruinssoftworks-s1-2560x1440-76fd08cd9c2d-2560x1440-783424f6d4a7f32a0e75fc074f2f928e.jpg'
                },
                {
                    type: 'OfferImageTall',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                },
                {
                    type: 'Thumbnail',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                },
                {
                    type: 'CodeRedemption_340x440',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                },
                {
                    type: 'DieselStoreFrontTall',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                }
            ],
            seller: {
                id: 'o-p9883lvrpllhyd7sad6tgxv53mdyh5',
                name: 'Hypetrain Digital'
            },
            productSlug: 'breathedge',
            urlSlug: 'osmiumgeneralaudience',
            url: null,
            items: [
                {
                    id: 'a07c7f65867e435dac277e0284e19a5a',
                    namespace: '08ae29e4f70a4b62aa055e383381aa82'
                }
            ],
            customAttributes: [
                {
                    key: 'publisherName',
                    value: 'HypeTrain Digital'
                },
                {
                    key: 'developerName',
                    value: 'RedRuins Softworks'
                },
                {
                    key: 'com.epicgames.app.productSlug',
                    value: 'breathedge'
                }
            ],
            categories: [
                {
                    path: 'freegames'
                },
                {
                    path: 'games'
                },
                {
                    path: 'games/edition'
                },
                {
                    path: 'games/edition/base'
                },
                {
                    path: 'applications'
                }
            ],
            tags: [
                {
                    id: '1216'
                },
                {
                    id: '1281'
                },
                {
                    id: '22737'
                },
                {
                    id: '21894'
                },
                {
                    id: '1080'
                },
                {
                    id: '1370'
                },
                {
                    id: '9547'
                },
                {
                    id: '9549'
                }
            ],
            catalogNs: {
                mappings: [
                    {
                        pageSlug: 'breathedge',
                        pageType: 'productHome'
                    }
                ]
            },
            offerMappings: [],
            price: {
                totalPrice: {
                    discountPrice: 2499,
                    originalPrice: 2499,
                    voucherDiscount: 0,
                    discount: 0,
                    currencyCode: 'USD',
                    currencyInfo: {
                        decimals: 2
                    },
                    fmtPrice: {
                        originalPrice: '$24.99',
                        discountPrice: '$24.99',
                        intermediatePrice: '$24.99'
                    }
                },
                lineOffers: [
                    {
                        appliedRules: []
                    }
                ]
            },
            promotions: {
                promotionalOffers: [],
                upcomingPromotionalOffers: [
                    {
                        promotionalOffers: [
                            {
                                startDate: '2023-04-27T15:00:00.000Z',
                                endDate: '2023-05-04T15:00:00.000Z',
                                discountSetting: {
                                    discountType: 'PERCENTAGE',
                                    discountPercentage: 0
                                }
                            }
                        ]
                    }
                ]
            }
        },
        invalid: {
            someRandomKey: 'xd'
        },
        other: {
            title: 'Breathedge',
            id: '8401414902e84f2cb9afa9142f051d32',
            namespace: '08ae29e4f70a4b62aa055e383381aa82',
            description: 'Survive in outer space! Together with your immortal chicken, discover the truth behind your sudden spaceship crash. Craft tools, pilot vehicles, and even control space stations to survive and explore the wreckage.',
            effectiveDate: '2021-02-25T18:00:00.000Z',
            offerType: 'OTHERS',
            expiryDate: null,
            viewableDate: '2021-02-09T17:00:00.000Z',
            status: 'ACTIVE',
            isCodeRedemptionOnly: false,
            keyImages: [
                {
                    type: 'OfferImageWide',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/egs-breathedge-redruinssoftworks-s1-2560x1440-76fd08cd9c2d-2560x1440-783424f6d4a7f32a0e75fc074f2f928e.jpg'
                },
                {
                    type: 'DieselStoreFrontWide',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/egs-breathedge-redruinssoftworks-s1-2560x1440-76fd08cd9c2d-2560x1440-783424f6d4a7f32a0e75fc074f2f928e.jpg'
                },
                {
                    type: 'OfferImageTall',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                },
                {
                    type: 'Thumbnail',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                },
                {
                    type: 'CodeRedemption_340x440',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                },
                {
                    type: 'DieselStoreFrontTall',
                    url: 'https://cdn1.epicgames.com/08ae29e4f70a4b62aa055e383381aa82/offer/EGS_Breathedge_RedRuinsSoftworks_S2-1200x1600-c0559585221ea11c9d48273c3a79b1ba.jpg'
                }
            ],
            seller: {
                id: 'o-p9883lvrpllhyd7sad6tgxv53mdyh5',
                name: 'Hypetrain Digital'
            },
            productSlug: 'breathedge',
            urlSlug: 'osmiumgeneralaudience',
            url: null,
            items: [
                {
                    id: 'a07c7f65867e435dac277e0284e19a5a',
                    namespace: '08ae29e4f70a4b62aa055e383381aa82'
                }
            ],
            customAttributes: [
                {
                    key: 'publisherName',
                    value: 'HypeTrain Digital'
                },
                {
                    key: 'developerName',
                    value: 'RedRuins Softworks'
                },
                {
                    key: 'com.epicgames.app.productSlug',
                    value: 'breathedge'
                }
            ],
            categories: [
                {
                    path: 'freegames'
                },
                {
                    path: 'games'
                },
                {
                    path: 'games/edition'
                },
                {
                    path: 'games/edition/base'
                },
                {
                    path: 'applications'
                }
            ],
            tags: [
                {
                    id: '1216'
                },
                {
                    id: '1281'
                },
                {
                    id: '22737'
                },
                {
                    id: '21894'
                },
                {
                    id: '1080'
                },
                {
                    id: '1370'
                },
                {
                    id: '9547'
                },
                {
                    id: '9549'
                }
            ],
            catalogNs: {
                mappings: [
                    {
                        pageSlug: 'breathedge',
                        pageType: 'productHome'
                    }
                ]
            },
            offerMappings: [],
            price: {
                totalPrice: {
                    discountPrice: 2499,
                    originalPrice: 2499,
                    voucherDiscount: 0,
                    discount: 0,
                    currencyCode: 'USD',
                    currencyInfo: {
                        decimals: 2
                    },
                    fmtPrice: {
                        originalPrice: '$24.99',
                        discountPrice: '$24.99',
                        intermediatePrice: '$24.99'
                    }
                },
                lineOffers: [
                    {
                        appliedRules: []
                    }
                ]
            },
            promotions: {
                promotionalOffers: [],
                upcomingPromotionalOffers: [
                    {
                        promotionalOffers: [
                            {
                                startDate: '2023-04-27T15:00:00.000Z',
                                endDate: '2023-05-04T15:00:00.000Z',
                                discountSetting: {
                                    discountType: 'PERCENTAGE',
                                    discountPercentage: 0
                                }
                            }
                        ]
                    }
                ]
            }
        }
    }
};
